db/utility.js

```diff
@@ -25,10 +25,35 @@ const handleComment = (itemTransResult, itemCommentResult, commentResult)=>{
 
 
 const queryAllItem = async()=>{
-    let itemTransResult = await dbQuery('SELECT * FROM `Item_Trans` WHERE `lang`=\'zh\'')
-    let itemResult = await dbQuery('SELECT * FROM `Item`')
-    let commentResult = await dbQuery('SELECT * FROM `Comment`')
-    let itemCommentResult = await dbQuery('SELECT * FROM `Item_Comment`')
+    let itemTransResult
+    let itemResult
+    let itemCommentResult
+    let commentResult
+
+    for(let i=0; i<500; i++){
+        itemTransResult = await dbQuery('SELECT * FROM `Item_Trans` WHERE `lang`=\'zh\'')
```

Comment:
why do you need to request items in a loop? **SELECT * FROM**  already returns a list of items

Reply:
I remove it.